documentation/source/newsfragments/1843.change.rst

```diff
@@ -0,0 +1 @@
+Simulator Python module now throws an exception if used when no simulator is present. Making the use of cocotb without a simulator present safe.
```

Comment:
```suggestion
The :mod:`cocotb.simulator` Python module now throws an exception if used when no simulator is present, making the use of cocotb without a simulator present safe.
```
(Cannot test right now if this will result in the correct link.)